src/structures/GuildMember.js

```diff
@@ -77,10 +71,19 @@ class GuildMember extends Base {
 
   _clone() {
     const clone = super._clone();
-    clone.roles = this.roles.clone();
+    clone._roles = this._roles;
     return clone;
   }
 
+  /**
+   * A list of roles that are applied to this GuildMember, mapped by the role ID
```

Comment:
I know the string was like that already, but a `list` sounds more like an `Array` than `Collection` or even `GuildMemberRoleStore`.